rever.xsh

```diff
@@ -1,7 +1,7 @@
 $ACTIVITIES = ["authors", "changelog"]
 
 # Basic settings
-$PROJECT = $GITHUB_REPO = $(basename $(git remote get-url origin)).split('.')[0]
+$PROJECT = $GITHUB_REPO = $(basename $(git remote get-url origin)).split('.')[0].strip()
```

Comment:
This removes the unwanted newline in `AUTHORS.md` ```
"All of the people who have made at least one contribution to conda

."
```

Reply:
This would need to be applied upstream in https://github.com/conda/infrastructure/blob/main/.github/sync/rever.xsh